android/java/res/drawable/ic_spot_graphic.xml

```diff
@@ -1,5 +1,7 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:aapt="http://schemas.android.com/aapt"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
i have been meaning to ask design team to provide vector file with suggested dimensions. and if we ignore it here, we would miss it. @SergeyZhukovsky what do you think ? it's the resources for wallet

Reply:
@deeppandya do you propose to leave all graphic as warnings for now?

Reply:
@SergeyZhukovsky i would suggest to leave it for now. i will reach out to James and ask for appropriate assets.

Reply:
ok, @wchen342 let's leave assets with `tools:ignore="VectorRaster"` with warning for now as well